.pre-commit-config.yaml

```diff
@@ -524,6 +524,7 @@ repos:
         language: system
         entry: "./scripts/ci/pre_commit/pre_commit_mypy.sh"
         files: ^docs/.*\.py$
+        exclude: rtd-deprecatipn
```

Comment:
Since we have two conf.py files and no __init__.py, mypy thinks we have two modules with the same name.